libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -1574,7 +1565,7 @@ - (void) testBlocks {
     self.currentExpectation = [self expectationWithDescription:@"performSOSLSearch-find {batman}"];
     [api performSOSLSearch:@"find {batman}"
                  failBlock:failWithUnexpectedFail
-             completeBlock:arraySuccessBlock];
+             completeBlock:dictSuccessBlock];
```

Comment:
How was that test passing for the last year?

Reply:
I'm not sure it was passing - our CI has been broken for a while now.

Reply:
It was passing locally.
I think because we didn't send any message to (i.e. call any method on) the array object in `arraySuccessBlock`, nothing bad happened.
Could you add an assert inside `dictSuccessBlock` that checks that `[d isKindOfClass:[NSDictionary class]]` ?

Reply:
Done [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/2411/commits/118de98f83558e99e5e7c4d60a821acb1d6b7f1d).